packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -198,6 +199,9 @@ const DefaultSummaryConfiguration: ISummaryConfiguration = {
     maxAckWaitTime: 120000,
 };
 
+// This is the current version of garbage collection.
+const GCVersion = 1;
```

Comment:
nit: should this be camelCase?
```suggestion
const gcVersion = 1;
```

Reply:
Other globals in this file start with upper case so I kept that.